lib/scarpe/button.rb

```diff
@@ -1,11 +1,46 @@
 # frozen_string_literal: true
 
+# The Scarpe Class
 class Scarpe
+  # The Button class represents a button widget in Scarpe
   class Button < Scarpe::Widget
     include Scarpe::Log
-
+    
     display_properties :text, :width, :height, :top, :left, :size
 
+    # Initializes a new Button object.
```

Comment:
Nice! This is much better documentation than we've been including. I'll keep working to get it put up where people can see it :-)